devwilliamy/coverland
# Added new My Orders page to User Profile (v1)

-Version One
-Click Up : https://app.clickup.com/t/8688pw53e

Figma:
Desktop - https://www.figma.com/design/f0PiJaMbj3moDGXq45iAA1/Coverland-Website-Design?node-id=4577-55056&t=2AMI64xIbxLwMP81-0
Mobile - https://www.figma.com/design/f0PiJaMbj3moDGXq45iAA1/Coverland-Website-Design?node-id=4619-68720&t=2AMI64xIbxLwMP81-0

*Design Notes: Just the orders from the My Account page
** Also Ashley is currently changing some of the design -> that will be rolled out in v2

-New page under /profile/orders
-My Orders returns 4 most recent user orders and displays a list of them with basic details

Test Instructions:
-sign into a user
-go to your profile /profile
-button "My Orders" -> navigates user to /profile/orders
-should output a list of up to 4 most recent user orders (if none, should indicate so)
-should redirect user to /login if they are not logged in at their orders page
